src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -317,8 +335,15 @@ class DropdownAutoCompleteMenu extends React.Component {
           // emptyHidesInput is set to true.
           let showInput = hasItems || !emptyHidesInput;
 
+          let renderProps = {actions, components: {LabelWithPadding}};
```

Comment:
is it worth it to do this, or should we just extract this stuff to a new component?

Reply:
Yeah this might just be better off as an importable component.